projects/sandbox/src/app/wizard/wizard.component.ts

```diff
@@ -57,37 +82,46 @@ export class WizardComponent extends DataPollingComponent {
    **/
   buildForms(): void {
     this.formProgress = this.fb.group({
-      idToTrack: ['', Validators.required]
+      idToTrack: ['', [Validators.required, this.validatorWhitespace]]
     });
-    this.formUpload = this.fb.group(
-      this.wizardConf
-        .filter((conf) => {
-          return !!conf.fields;
-        })
-        .reduce((map: ValidatorArrayHash, conf: WizardStep) => {
-          const test = (conf.fields ? conf.fields : [])
-            .filter((map: WizardField) => !!map.name)
-            .reduce((map: ValidatorArrayHash, wf: WizardField) => {
-              const entry: Array<string | Validators[]> = [wf.defaultValue ? wf.defaultValue : ''];
-              if (wf.validators) {
-                entry.push(wf.validators);
-              }
-              map[wf.name] = entry;
-              return map;
-            }, {});
-          return Object.assign(map, test);
-        }, {})
-    );
+
+    this.formUpload = this.fb.group({
+      name: ['', [Validators.required, this.validatorWhitespace]],
```

Comment:
I think the name requires some more validation. Will this be part of a separate subsequent ticket?

Reply:
Yes - full validation would have been out of scope for this ticket!